vue-client/src/components/inspector/item-local.vue

```diff
@@ -559,7 +603,7 @@ export default {
         />
 
         <entity-action
-          v-if="inspector.status.editing && !isLocked"
+          v-if="inspector.status.editing && !isLocked && !isInlinedRecord && !isExtracting"
```

Comment:
`!isExtracting` is a workaround for a bug in the old "extract" code.
I you start extracting an entity and the delete it you get an error when saving.

It is still possible to trigger:
* Add a new Item in has component
* Click "Extract Single item" / "Extrahera Exemplar"
* Click Undo (removing the added item)
* Save
* Error!